src/sass/card/stories/card-large.stories.js

```diff
@@ -0,0 +1,130 @@
+export default { title: "Skin/Card/Large" };
+
+export const base = () => `
+    <div class="card card--large">
+        <div class="card__container card__container--large">
```

Comment:
`card__container--large` seems unnecessary since it is _always_ wrapped in `card--large`, maybe use a cascade instead?

Reply:
good call, will fix